components/org.wso2.analytics.is/src/main/test/org/wso2/analytics/is/siddhi/extension/EventDuplicateStreamProcessorTestCase.java

```diff
@@ -43,7 +43,6 @@ public void init() {
 
     @Test
     public void eventDuplicatorDefaultDelimiterTest1() throws InterruptedException {
-
         SiddhiManager siddhiManager = new SiddhiManager();
```

Comment:
Why these lines added for readability are removed

Reply:
Added it back.